conda_forge_tick/update_upstream_versions.py

```diff
@@ -9,11 +9,63 @@
 from conda.models.version import VersionOrder
 from pkg_resources import parse_version
 
-from .utils import parse_meta_yaml, setup_logger
+from utils import parse_meta_yaml, setup_logger
+import os
 
 logger = logging.getLogger("conda_forge_tick.update_upstream_versions")
 
 
+class LibrariesIO:
+
+    def __init__(self):
+        self.libraries_io_api_key = os.getenv("LIBRARIES_IO_API_KEY")
+        self._data = None
+
+    def get_libraries_io_subscriptions(self):
+        if self.libraries_io_api_key == None:
+            return
+        l = []
+        for i in range(1, 2):
+            url = "https://libraries.io/api/subscriptions?api_key={}&page={}&per_page={}&include_prerelease=False"
+            url = url.format(self.libraries_io_api_key, i, 100)
+            print(url.replace(self.libraries_io_api_key, "<dummy>"))
+            r = retry_requests(requests.get, url)
+            if not r.ok or len(r.json()) == 0:
+                break
+            l.extend(r.json())
+        return l
+
+    @property
+    def data(self):
+        if self._data is not None:
+            return self._data
+        d = {'pypi':{}, 'cran':{}}
```

Comment:
Maybe use `d = defaultdict(dict)`. This would make it easier to implement other LibrariesIO package managers in the future.